test/test-poll-multiple-handles.c

```diff
@@ -0,0 +1,84 @@
+/* Copyright Joyent, Inc. and other Node contributors. All rights reserved.
```

Comment:
```suggestion
/* Copyright libuv project contributors. All rights reserved.
```
(new files should use the new copyright)